src/types/Navigation.ts

```diff
@@ -48,6 +48,7 @@ export enum ScreenName {
   OParlOverview = 'OParlOverview',
   OParlPersons = 'OParlPersons',
   OParlSearch = 'OParlSearch',
+  SecretIcons = 'SecretIcons',
```

Comment:
let's use a prefix for Icons depending on the path structure we chose, `DocIcons`/`DocumentationIcons`, `AboutIcons` or `HelpIcons`?

Reply:
adjusted the prefix matching with the new path structure [29186de](https://github.com/smart-village-solutions/smart-village-app-app/pull/914/commits/29186de714d420a15bf9284a4e17cbc553609203)

Reply:
DocIcons let me think of doctors somehow 👨‍⚕️ 👩‍⚕️ but all right, let's keep it!